packages/webview_flutter/webview_flutter/lib/src/webview.dart

```diff
@@ -260,6 +261,13 @@ class WebView extends StatefulWidget {
   /// By default `gestureNavigationEnabled` is false.
   final bool gestureNavigationEnabled;
 
+  /// A Boolean value that determines whether pressing a link displays a preview of the destination for the link.
+  ///
+  /// Not supported on all platforms.
+  ///
+  /// By default `allowsLinkPreview` is true, to match the default on iOS.
```

Comment:
Since this is the cross-platform API, I'd prefer that we not include the "to match the default on iOS" part (since if we added support for other platforms in the future that had different platform defaults, it wouldn't be clear to a reader why we match iOS rather than the other platforms.)

Reply:
Changed to "By default `allowsLinkPreview` is true."